### PR TITLE
Update exercise page.

### PR DESCRIPTION
**Pull request by eschlange**
* Added exit button link, navigates to home.
* Added save indication for exercises.

[#105108312] [#105108180]

### CODE REVIEW

File: src/partials/exercise.html [hunk 1]
```diff
@@ -17,6 +29,19 @@
     </div>
 </div>
 
+<script type="text/javascript">
+    $(document).ready (function() {
+        $("#success-alert").hide();
+        $("#save_button").click(function() {
+            debugger
+            $("#success-alert").show();
```
Comment (michaelwehrley) [inline src/partials/exercise.html]: I think you can chain all of this: 
```
$("#success-alert").show().fadeTo(2000, 500).slideUp(500, function(){
  $("#success-alert").hide();
});
```

File: src/partials/exercise.html [hunk 2]
```diff
@@ -17,6 +29,19 @@
     </div>
 </div>
 
+<script type="text/javascript">
+    $(document).ready (function() {
+        $("#success-alert").hide();
+        $("#save_button").click(function() {
+            debugger
```
Comment (michaelwehrley) [inline src/partials/exercise.html]: note.

File: src/partials/exercise.html [hunk 3]
```diff
@@ -9,7 +9,19 @@
                     ng-model="exercise.todaysExerciseAnswers[key].answer"
                     placeholder="{{ value.instruction }}"></textarea>
         </div>
-        <button id="save_button" class="btn btn-info wide" ng-click="exercise.saveExerciseAnswers()">Save</button><br/><br/>
+
+        <div class="alert alert-success" id="success-alert">
+            <button type="button" class="close" data-dismiss="alert">x</button>
+            <strong>save complete</strong>
```
Comment (michaelwehrley) [inline src/partials/exercise.html]: Does this need to be capitalized?

**Comment by eschlange** [comment]
@michaelwehrley Chaining does work, also removed debugger.

File: src/partials/exercise.html [hunk 4]
```diff
@@ -9,7 +9,19 @@
...
+        </div>
+
+        <a id="save_button"
+           class="btn btn-info"
+           ng-click="exercise.saveExerciseAnswers()">Save</a>
+        <a id="exit_button"
```
Comment (michaelwehrley) [inline src/partials/exercise.html]: just curious, but did you put an `id` here for testing?

File: src/partials/exercise.html (hunk 3 above)
Comment (eschlange) [inline src/partials/exercise.html]: no, non-functional requirement to keep it lower case.

File: src/partials/exercise.html (hunk 4 above)
Comment (eschlange) [inline src/partials/exercise.html]: I usually always put IDs on unique tags, force of habit.

**Comment by michaelwehrley** [comment]
Sounds good.  Thanks.

**Pull request merged**